src/USER-MANIFOLD/fix_nve_manifold_rattle.cpp

```diff
@@ -94,6 +94,8 @@ FixNVEManifoldRattle::FixNVEManifoldRattle( LAMMPS *lmp, int &narg, char **arg,
   dof_flag = 1;
 
   nevery = 0;
+  next_output = 0;
+  fprintf( screen, "nevery = %d\n", nevery );
```

Comment:
this looks like some leftover debug statement. i'll try to remove it through the github edit function.